custom-recipes/nlp-visualization-wordcloud/recipe.json

```diff
@@ -1,115 +1,129 @@
 {
-    "meta": {
-        "label": "Word clouds",
-        "description": "Generate word clouds from your text data",
-        "icon": "icon-comment"
+  "meta": {
```

Comment:
This recipe.json version is not the one from `main`. Can you merge main into your branch and solve the conflicts?